frontend/src/components/portforward/index.tsx

```diff
@@ -190,19 +189,31 @@ export default function PortForwardingList() {
                   <Box
                     mr={1}
                     onClick={() => {
+                      if (portforward.status === PORT_FORWARD_STOP_STATUS) {
+                        return;
+                      }
```

Comment:
This doesn't disable the hint that we can click the link. There should be another way to disable the link.